android/app/src/main/kotlin/coop/polypoly/polypod/PushNotificationWorker.kt

```diff
@@ -11,8 +11,7 @@ import androidx.core.app.NotificationCompat
 import androidx.core.app.NotificationManagerCompat
 import androidx.work.Worker
 import androidx.work.WorkerParameters
-import coop.polypoly.polypod.MainActivity
-import coop.polypoly.polypod.R
+import coop.polypoly.polypod.core.UpdateNotification
```

Comment:
Again, you've moved those two little buddies to this package, right?

Reply:
`PushNotificationWorker` itself moved to the `coop.polypoly.polypod` package - so it doesn't need to include its new siblings (`MainActivity` and `R`), but it _does_ now need to include its former sibling, `UpdateNotification`.